pom.xml

```diff
@@ -11,7 +11,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>23.0</version>
+      <version>15.0</version>
```

Comment:
Curious what happened here. Are you running an older version of Java?

Reply:
IntelliJ found this version. Did not bother investigating further.